src/apps/SW4CK_KERNEL_5-Seq.cpp

```diff
@@ -0,0 +1,208 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2017-23, Lawrence Livermore National Security, LLC
+// and RAJA Performance Suite project contributors.
+// See the RAJAPerf/LICENSE file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+
+#include "SW4CK_KERNEL_5.hpp"
+
+#include "RAJA/RAJA.hpp"
+
+#include "AppsData.hpp"
+
+#include <iostream>
+
+namespace rajaperf
+{
+namespace apps
+{
+
+
+void SW4CK_KERNEL_5::runSeqVariant(VariantID vid, size_t RAJAPERF_UNUSED_ARG(tune_idx))
+{
+  const Index_type run_reps = getRunReps();
+
+  //To be populated later with
+  const int istart = 0;
+  const int ifirst = 0;
+  const int ilast = 1;
+  const int jstart = 0;
+  const int jfirst = 0;
+  const int jend = 0;
+  const int jlast = 1;
+  const int kfirst = 0;
+  const int kstart = 0;
+  const int klast = 1;
+  const int kend = 1;
```

Comment:
@Ram3sh, looking at the SW4CK src I think I see various options for problem size dimensions. Would you be able to advise what the values here should be? Or some values that we can use to scale the problem?